src/UserManager.ts

```diff
@@ -332,7 +332,7 @@ export class UserManager {
     protected async _useRefreshToken(args: UseRefreshTokenArgs): Promise<User> {
         const response = await this._client.useRefreshToken({
             ...args,
-            timeoutInSeconds: this.settings.silentRequestTimeoutInSeconds,
+            timeoutInSeconds: this.settings.requestTimeoutInSeconds ?? this.settings.silentRequestTimeoutInSeconds,
```

Comment:
There are more places where `silentRequestTimeoutInSeconds` (e.g. `IFrameNavigator`) is used. I guess it would be better to handle that in `UserManagerSettingsStore`:

Idea if requestTimeoutInSeconds  is set we use the same value for `silentRequestTimeoutInSeconds`

something like
```
this.silentRequestTimeoutInSeconds = requestTimeoutInSeconds ?? silentRequestTimeoutInSeconds,
```